build.gradle

```diff
@@ -30,8 +30,8 @@ dependencies {
   implementation "com.amazonaws:aws-java-sdk-s3:$awsSDKVersion"
   implementation "org.apache.commons:commons-lang3:$apacheCommonsVersion"
   implementation "org.rogach:scallop_$scalaVersion:$scallopVersion"
-  implementation "com.microsoft.sqlserver:mssql-jdbc:$sqlserverVersion"
-  implementation "org.postgresql:postgresql:$postgresqlVersion"
+  compile "com.microsoft.sqlserver:mssql-jdbc:$sqlserverVersion"
```

Comment:
Why is this necessary? I believe that `compile` is deprecated.

Reply:
What we found is that if you leave it as "implementation" the code doesn't pick up the correct driver at run time - it has to be installed separately and managed. Especially true with PostgreSQL.  Changing to "compile" fixes that problem.
I have warnings for deprecated items turned on in IntelliJ - it isn't reporting any issues.

Reply:
Compile is deprecated in Gradle 6: https://github.com/gradle/gradle/issues/15661